src/components/TypeBox/index.tsx

```diff
@@ -33,7 +35,11 @@ const TypeBox = ({
 
   const router = useRouter();
 
-  let words: any = [];
+  const getDataset = (): WordsDataset => {
+    return datasetLanguage === "es" ? esDataset : enDataset;
```

Comment:
Si pensamos agregar mas dataSets en un futuro capaz podriamos implementar un diccionario de esta manera
```suggestion
    const dataSets = {
     "es":esDataset,
     "en":enDataset
    }
    return dataSets[datasetLanguage];
```